[0m16:45:31.046676 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e72be4590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e70c42850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e70c42e90>]}


============================== 16:45:31.056195 | de1b5803-10c6-4e1b-af18-5ce7cebce80c ==============================
[0m16:45:31.056195 [info ] [MainThread]: Running with dbt=1.8.2
[0m16:45:31.057033 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/usr/app', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m16:45:55.045693 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.13/site-packages/dbt/include/starter_project
[0m16:45:55.315506 [info ] [MainThread]: 
Your new dbt project "sales_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:45:55.316114 [info ] [MainThread]: Setting up your profile.
[0m16:46:58.812170 [info ] [MainThread]: Profile sales_project written to /usr/app/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:46:58.816074 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 88.16887, "process_user_time": 1.01783, "process_kernel_time": 0.191821, "process_mem_max_rss": "97528", "process_in_blocks": "114080", "process_out_blocks": "1704"}
[0m16:46:58.816846 [debug] [MainThread]: Command `dbt init` succeeded at 16:46:58.816713 after 88.17 seconds
[0m16:46:58.817473 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e706b1940>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e711df650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x787e70e87460>]}
[0m16:46:58.818381 [debug] [MainThread]: Flushing usage events
[0m16:47:10.729391 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d87101a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d8bcf890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d696e350>]}


============================== 16:47:10.732503 | e73dad12-b965-436c-a03d-01e1f4414175 ==============================
[0m16:47:10.732503 [info ] [MainThread]: Running with dbt=1.8.2
[0m16:47:10.733330 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/usr/app', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:47:10.749832 [info ] [MainThread]: dbt version: 1.8.2
[0m16:47:10.750619 [info ] [MainThread]: python version: 3.13.7
[0m16:47:10.751531 [info ] [MainThread]: python path: /usr/local/bin/python3.13
[0m16:47:10.752335 [info ] [MainThread]: os info: Linux-6.6.87.2-microsoft-standard-WSL2-x86_64-with-glibc2.41
[0m16:47:10.798693 [info ] [MainThread]: Using profiles dir at /usr/app
[0m16:47:10.799373 [info ] [MainThread]: Using profiles.yml file at /usr/app/profiles.yml
[0m16:47:10.799886 [info ] [MainThread]: Using dbt_project.yml file at /usr/app/dbt_project.yml
[0m16:47:10.801572 [info ] [MainThread]: adapter type: postgres
[0m16:47:10.802503 [info ] [MainThread]: adapter version: 1.8.2
[0m16:47:10.803633 [info ] [MainThread]: Configuration:
[0m16:47:10.804263 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m16:47:10.804996 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m16:47:10.805951 [info ] [MainThread]: Required dependencies:
[0m16:47:10.806901 [debug] [MainThread]: Executing "git --help"
[0m16:47:10.822121 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m16:47:10.822797 [debug] [MainThread]: STDERR: "b''"
[0m16:47:10.823319 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m16:47:10.824048 [info ] [MainThread]: Connection:
[0m16:47:10.824919 [info ] [MainThread]:   host: postgres
[0m16:47:10.826102 [info ] [MainThread]:   port: 5432
[0m16:47:10.826797 [info ] [MainThread]:   user: postgres
[0m16:47:10.827416 [info ] [MainThread]:   database: sales
[0m16:47:10.827945 [info ] [MainThread]:   schema: DW_sales
[0m16:47:10.828434 [info ] [MainThread]:   connect_timeout: 10
[0m16:47:10.828968 [info ] [MainThread]:   role: None
[0m16:47:10.829535 [info ] [MainThread]:   search_path: None
[0m16:47:10.830223 [info ] [MainThread]:   keepalives_idle: 0
[0m16:47:10.830907 [info ] [MainThread]:   sslmode: None
[0m16:47:10.831599 [info ] [MainThread]:   sslcert: None
[0m16:47:10.832123 [info ] [MainThread]:   sslkey: None
[0m16:47:10.832628 [info ] [MainThread]:   sslrootcert: None
[0m16:47:10.833078 [info ] [MainThread]:   application_name: dbt
[0m16:47:10.833575 [info ] [MainThread]:   retries: 1
[0m16:47:10.834334 [info ] [MainThread]: Registered adapter: postgres=1.8.2
[0m16:47:10.884574 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m16:47:10.920143 [debug] [MainThread]: Using postgres connection "debug"
[0m16:47:10.921051 [debug] [MainThread]: On debug: select 1 as id
[0m16:47:10.921766 [debug] [MainThread]: Opening a new connection, currently in state init
[0m16:47:10.959007 [debug] [MainThread]: SQL status: SELECT 1 in 0.037 seconds
[0m16:47:10.960160 [debug] [MainThread]: On debug: Close
[0m16:47:10.960798 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m16:47:10.961260 [info ] [MainThread]: [31m1 check failed:[0m
[0m16:47:10.961754 [info ] [MainThread]: Project loading failed for the following reason:
 project path </usr/app/dbt_project.yml> not found

[0m16:47:10.963106 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.28935748, "process_user_time": 0.956259, "process_kernel_time": 0.103164, "process_mem_max_rss": "104696", "process_in_blocks": "5872", "process_out_blocks": "32", "command_success": false}
[0m16:47:10.963733 [debug] [MainThread]: Command `dbt debug` failed at 16:47:10.963613 after 0.29 seconds
[0m16:47:10.964171 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m16:47:10.964567 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d532d5b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d5fe17f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7242d5f74f30>]}
[0m16:47:10.965015 [debug] [MainThread]: Flushing usage events
[0m15:51:31.532108 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b149588590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b149a47c50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b14780a850>]}


============================== 15:51:31.541693 | f72a9449-f8df-4935-bc8e-6b65ac72cc4f ==============================
[0m15:51:31.541693 [info ] [MainThread]: Running with dbt=1.8.2
[0m15:51:31.542560 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/usr/app', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt test', 'send_anonymous_usage_stats': 'True'}
[0m15:51:31.544671 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /usr/app/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:51:31.546708 [debug] [MainThread]: Resource report: {"command_name": "test", "command_wall_clock_time": 0.16153367, "process_user_time": 1.074557, "process_kernel_time": 0.215792, "process_mem_max_rss": "94416", "process_in_blocks": "103032", "process_out_blocks": "1704", "command_success": false}
[0m15:51:31.547542 [debug] [MainThread]: Command `dbt test` failed at 15:51:31.547399 after 0.16 seconds
[0m15:51:31.565651 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b147723490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b1476144d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76b1476a0050>]}
[0m15:51:31.566406 [debug] [MainThread]: Flushing usage events
[0m16:00:00.475868 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cfaccc1a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cfb18b890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cf8f2a350>]}


============================== 16:00:00.479135 | 30e1ca1c-db30-47e2-a0da-282c69719a0d ==============================
[0m16:00:00.479135 [info ] [MainThread]: Running with dbt=1.8.2
[0m16:00:00.479985 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/usr/app', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt build', 'send_anonymous_usage_stats': 'True'}
[0m16:00:00.481389 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /usr/app/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:00:00.482745 [debug] [MainThread]: Resource report: {"command_name": "build", "command_wall_clock_time": 0.061606824, "process_user_time": 0.948248, "process_kernel_time": 0.068017, "process_mem_max_rss": "92924", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m16:00:00.483510 [debug] [MainThread]: Command `dbt build` failed at 16:00:00.483368 after 0.06 seconds
[0m16:00:00.484038 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cf8fab100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cf90eb9b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721cf8efbdf0>]}
[0m16:00:00.484601 [debug] [MainThread]: Flushing usage events
[0m12:41:00.159648 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e1037c590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e1084bc50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e0e3f7d90>]}


============================== 12:41:00.177105 | 4bf98b88-290d-4990-9edd-98d548da29f2 ==============================
[0m12:41:00.177105 [info ] [MainThread]: Running with dbt=1.8.2
[0m12:41:00.178772 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/usr/app', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt docs generate', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:41:00.190206 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /usr/app/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:41:00.193734 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_wall_clock_time": 0.2145812, "process_user_time": 2.499942, "process_kernel_time": 0.377508, "process_mem_max_rss": "94160", "process_in_blocks": "103032", "process_out_blocks": "1704", "command_success": false}
[0m12:41:00.195592 [debug] [MainThread]: Command `dbt docs generate` failed at 12:41:00.195379 after 0.22 seconds
[0m12:41:00.196572 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e0e54b5c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e0e43c4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x777e0e4ac050>]}
[0m12:41:00.197633 [debug] [MainThread]: Flushing usage events
